spec/api-net-spec.js

```diff
@@ -1400,11 +1399,8 @@ describe('net module', () => {
         assert(typeof httpVersionMinor === 'number')
         assert(httpVersionMinor >= 0)
         response.pause()
-        response.on('data', (chunk) => {
-        })
-        response.on('end', () => {
-          done()
-        })
+        response.on('data', (chunk) => {})
+        response.on('end', () => { done() })
         response.resume()
       })
       urlRequest.end()
```

Comment:
Would like a specific test for the new deduping behaviour

Reply:
done!